facebook/fboss
# janga: config: updated platform manager config to support respin board

**Description**

Updated the platform manager config to support 2nd source based on 'VersionedPmUnitConfig' in fboss with latest source code.
Sync up config from BSP repo, https://github.com/facebookexternal/fboss.bsp.celestica/pull/443

**Motivation**

1. 1. Added 2nd source config with two interchangeable chips based on 'VersionedPmUnitConfig'.
2. ![image](https://github.com/user-attachments/assets/6bb4f340-9830-4959-8dd0-253da7d98b10)
3. ![image](https://github.com/user-attachments/assets/351835b4-b62d-49ff-bbb9-ed1a97212496)
4. 
5. removed the fan watchdog with requirement, detial info reference PR:https://github.com/facebookexternal/fboss.bsp.celestica/pull/460

**Test Plan**

1. build the latest fboss bsp kmod.
2. build the latest fboss platform manager binary.
3. Run the platform manager with the changed config files on DVT and 2nd source DVT device normally.
4. Run fw_util check unit version information.
5. Run the weutil check the unit eeprom data and product sub version.
IFX part chips board tested log:
[Janga-replace_parts_IFX_platform_manager_log_20240815.txt](https://github.com/user-attachments/files/17000982/Janga-replace_parts_IFX_platform_manager_log_20240815.txt)

MPS part chips board tested log:
[Janga-replace_parts_MPS_platform_manager_log_20240815.txt](https://github.com/user-attachments/files/17000983/Janga-replace_parts_MPS_platform_manager_log_20240815.txt)
